<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="BINCDV-I10165" name="Regression.US.US_Approved_Email_Multichannel_Activity" tests="1" failures="1" timestamp="14 Jul 2017 13:59:21 GMT" time="179.930" errors="0">
  <testcase name="US_Approved_Email_Multichannel_Activity_Validation" time="179.930" classname="Regression.US.US_Approved_Email_Multichannel_Activity">
    <failure type="java.lang.AssertionError" message="Test Case failed. ErrorMsg:  Test Case failed. ErrorMsg: Exception caught while executing verifyElementFound: Element xpath=//*[@class=&#039;UISlider&#039;] wasn&#039;t found">
      <![CDATA[java.lang.AssertionError: Test Case failed. ErrorMsg:  Test Case failed. ErrorMsg: Exception caught while executing verifyElementFound: Element xpath=//*[@class='UISlider'] wasn't found
	at org.testng.Assert.fail(Assert.java:94)
	at com.cognizant.framework.Report.updateTestLog(Report.java:171)
	at com.framework.testbase.TestBase.testFailed(TestBase.java:450)
	at Regression.US.US_Approved_Email_Multichannel_Activity.US_Approved_Email_Multichannel_Activity_Validation(US_Approved_Email_Multichannel_Activity.java:181)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- US_Approved_Email_Multichannel_Activity_Validation -->
</testsuite> <!-- Regression.US.US_Approved_Email_Multichannel_Activity -->
